### PR TITLE
[docs] Spanish translation of attention.md

### PR DESCRIPTION
**Pull request by aaronjimv**
# What does this PR do?

Add the Spanish version of **`attention.md`** to `transformers/docs/source/es`

#28936

Fixes # (issue)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#create-a-pull-request),
      Pull Request section?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

@stevhliu

### CODE REVIEW

**Comment by aaronjimv** [comment]
cc: @tadeodonegana. @gisturiz

Hello guys, I would appreciate if you can review the translation. Thank you.

**Review by stevhliu — APPROVED** [review]
Thanks, it LGTM from my end! We can merge once the content has been reviewed 🤗

File: docs/source/es/attention.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+specific language governing permissions and limitations under the License.
+
+⚠️ Note that this file is in Markdown but contain specific syntax for our doc-builder (similar to MDX) that may not be
+rendered properly in your Markdown viewer.
+
+-->
+
+# Mecanismos de atención
+
+La mayoría de los modelos de transformadores utilizan atención completa, en el sentido de que la matriz de atención es cuadrada. Esto puede ser un gran cuello de botella computacional cuando tienes textos largos. `Longformer` y `reformer` son modelos que intentan ser más eficientes y utilizan una versión dispersa de la matriz de atención para acelerar el entrenamiento.
```
Comment (tadeodonegana) [inline docs/source/es/attention.md:19]: When writing about transformers (specifically in the phrase "modelos de transformadores"), I believe that using 'transformadores' may not be the most accurate translation. In my opinion, it is preferable to consistently refer to them as "transformers," regardless of the language being used.

**Review by gisturiz — APPROVED** [review]
LGTM!

File: docs/source/es/attention.md (hunk 1 above)
Comment (tadeodonegana) [reply docs/source/es/attention.md:19]: @aaronjimv @stevhliu I just added this comment. I'm not sure if it's correct, but I wanted to express my thoughts on this.

Comment (aaronjimv) [reply docs/source/es/attention.md:19]: hi @tadeodonegana thanks, I thinks its a good point. I going to use to "`transformers`"

Comment (stevhliu) [reply docs/source/es/attention.md:19]: I think when referring to the library, it is good to refer to it as just Transformers (no translation), but when referring to the models in a more generic way then it is ok to translate it.

Comment (tadeodonegana) [reply docs/source/es/attention.md]: Sounds good to me! Happy to help reviewing future translations! Great work @aaronjimv!

**Pull request merged**